<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>itemUser</class>
 <widget class="QWidget" name="itemUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>45</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="w_infos" native="true">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>0</y>
     <width>355</width>
     <height>45</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QLabel" name="lbl_username">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>350</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NickName</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_note">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>12</y>
      <width>360</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;note&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_adress">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>25</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Adress</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lbl_image">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressources.qrc">:/images/defaultUser.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
